examples/chip-tool/commands/pairing/Commands.h

```diff
@@ -173,6 +173,16 @@ class PairSoftAP : public PairingCommand
     {}
 };
 
+#if CHIP_DEVICE_CONFIG_ENABLE_WIFIPAF
+class PairWiFiPAF : public PairingCommand
+{
+public:
+    PairWiFiPAF(CredentialIssuerCommands * credsIssuerConfig) :
+        PairingCommand("wifi-paf", PairingMode::WiFiPAF, PairingNetworkType::WiFi, credsIssuerConfig)
```

Comment:
PAF is a feature of Wi-Fi spec, which is to send the public action frame to advertise it's existence and exchanges the messages. The Wi-Fi only device can advertise itself from this. I will prefer using "wifipaf-wifi" if it's ok.
Could you let me know  which document should I reference for the rule of naming?
"code-wifi" is different since BLE is required

Reply:
code-wifi should scan on all network technologies (PAF, DNS-SD, BLE) and then commission onto wifi.

"wifipaf-wifi" would be OK in general, but code-wifi needs to work with PAF as well.

>  Could you let me know which document should I reference for the rule of naming?

There is no document, just the existing naming bits.

Reply:
The WiFiPAF has been added to code-wifi that we can use this method to pair with the device. Please review the latest patch

Reply:
Thank you.